content/rule_book/quiz_events.md

```diff
@@ -1,506 +1,193 @@
 
-### Scoring Terms and Definitions
+When a challenge is made, the quizmaster must give each team's captain or co-captain an opportunity to speak to the challenge before making a ruling on the challenge. Whichever team is being challenged against will have the right to speak last.
 
-**Quiz Event**
-: Label for whatever quiz event triggered the run of the procedure; possible values are: "question", "foul", "timeout", "sub-in", "sub-out", "challenge", "unreadiness", and "unsportsmanlike"
+## Protests
 
-**Quiz Type**
-: Current quiz type, defined under the *Quiz Process/Types of Quizzes* section
+Coaches, including assistant coaches, of the teams participating in a quiz may place the quiz under protest under the following conditions:
 
-**Current/Next Question Form**
-: Current or next question "form" such as: "Standard", "Toss-Up", and "Bonus"
+- When a coach feels a logistical rule has been broken by the quiz officials
+- When a coach feels a ruling on a challenge is incorrect
 
-**Current/Next Question Integer**
-: Current or next question core integer value; for example, if on question 17A, the value is 17
+The protest must apply to the question or ruling that just occurred. The protest must be placed before the next question's type is made known. If a protest is launched after question number 20 is completed it must be done immediately after the ruling is announced.
 
-**Current/Next Question Label**
-: Current or next question label, the possible suffix; for example, if on question 17A, the value is A
+The coach may only confer with their assistant coach and substitute quizzer, before protesting.
 
-**Current/Next Question Number**
-: Current or next question number, which is a combination of integer and label
+The quiz officials and quiz team coaches will meet privately and attempt to come to an agreement. If no agreement is reached within 10 minutes, the meet director will arbitrate and determine the just solution.
 
-**Ruling**
-: Ruling on a question; possible values are: "correct", "incorrect", and "none" (meaning no jump)
+### Example
 
-**Challenge**
-: Ruling on a challenge; possible values are: "accepted" and "overruled"
+As an example of a logistical rule, consider a scenario wherein the quizmaster did not allow the full question answer duration period to expire for the quizzer when the quizzer was in context and had not given any wrong information.
 
-**Overruled Challenges**
-: Integer representing total overruled challenges for the given team
+## Fouls
 
-**Quizzer Score Increment**
-: Amount the given quizzer's score should be incremented
+A foul is a penalty called by a quiz official against a quizzer or team for conduct that breaks the text of spirit of the rules.
 
-**Team Score Increment**
-: Amount the given team's score should be incremented
+- Fouls may be called by any quiz official actively involved in the current quiz.
+- A foul must be called before the quizmaster makes known next question's type.
```

Comment:
make consistent "before the next question's type is made known"

Reply:
deab639